lib/manager/jenkins/readme.md

```diff
@@ -1,13 +1,3 @@
-The Jenkins manager supports the following format of the plugin list:
+The Jenkins manager supports a custom text or yaml format of the plugin list as described [here](https://github.com/jenkinsci/plugin-installation-manager-tool#plugin-input-format). Currently, only versions from the main [update center](https://updates.jenkins.io/) are supported.
```

Comment:
```suggestion
The Jenkins manager supports a custom text or YAML format of the plugin list as described [here](https://github.com/jenkinsci/plugin-installation-manager-tool#plugin-input-format).
Currently, only versions from the main [update center](https://updates.jenkins.io/) are supported.
```

Fixing-up some small stuff:

- Use one sentence per line
- YAML is an abbreviation and should be capitalized

Reply:
Updated